Makefile.cache

```diff
@@ -249,9 +249,8 @@ define LOAD_FROM_CACHE
 		$(if $(LOAD_DRV_DEB), $($(1)_CACHE_USER) tar -C $($(1)_BASE_PATH) -mxzvf $(CACHE_FILE_SELECT) 1>> $($(1)_DST_PATH)/$(1).log ,echo );
 		echo "File $(CACHE_FILE_SELECT) is loaded from cache into $($(1)_BASE_PATH)" >> $($(1)_DST_PATH)/$(1).log
 		$(eval $(1)_CACHE_LOADED := Yes)
-		$(shell touch $(CACHE_FILE_SELECT))
+		$(if $(call CHECK_WCACHE_ENABLED,$(1)), $(shell touch $(CACHE_FILE_SELECT)))
```

Comment:
Could you please change to mount as readonly in Makefile.work?
DOCKER_RUN += -v "$(SONIC_DPKG_CACHE_SOURCE):/dpkg_cache:rw"

Reply:
No harm to keep it as `rw`.

---
In reply to: [449937306](https://github.com/Azure/sonic-buildimage/pull/4896#discussion_r449937306) [](ancestors = 449937306)